website/project/model.py

```diff
@@ -75,16 +74,9 @@ def has_anonymous_link(node, auth):
     :param str link: any view-only link in the current url
     :return bool anonymous: Whether the node is anonymous to the user or not
     """
-    view_only_link = auth.private_key or request.args.get('view_only', '').strip('/')
```

Comment:
tweaked in "[fix the old bug](f3beef9c537fe117f85a9e51c8b5af18a7d68ee5)"

moved in 87707d0fa382e5cd1a8d58e27dedfc51285ac502

got it! https://github.com/CenterForOpenScience/osf.io/commit/0de958fe85d12816b8e6594c3ec16bfaf6fc8726#diff-033d4b3df3102b7d1aace074938585e9R470